src/service.ts

```diff
@@ -3,7 +3,11 @@ import { ApiCheckService, DslConstructorInput } from "@useoptic/api-checks";
 import { oas } from "@stoplight/spectral-rulesets";
 
 export function newSnykApiCheckService() {
-  const snykRulesService = new ApiCheckService<SynkApiCheckContext>();
+  const snykRulesService = new ApiCheckService<SynkApiCheckContext>(
+    (context) => {
+      return new Date(context.changeDate);
```

Comment:
@cmars this is where that key assumption lives re: what date should be used during the comparison. 

If it's the resource date from vervet, this code is right

if it should be today's date, we would replace with `new Date()`

I'm pretty sure you want it as is, but lmk if I misunderstood

Reply:
Looks correct as it is, `changeDate` is the date the change should be considered to take place. It could be (and usually is) "right now" but it is useful to be able to control it so that you can set a policy around it, or check how a rule might evaluate in a given scenario. Tests for sure, but also perhaps to answer questions like "I want to do ${crazy_thing_to_the_api} next week, will it be allowed"?